core/src/display_object/avm1_button.rs

```diff
@@ -401,7 +401,16 @@ impl<'gc> TDisplayObject<'gc> for Avm1Button<'gc> {
 }
 
 impl<'gc> TDisplayObjectContainer<'gc> for Avm1Button<'gc> {
-    impl_display_object_container!(container);
+    fn raw_container(&self) -> Ref<'_, ChildContainer<'gc>> {
```

Comment:
Minor note: We use the same pattern for `TInteractiveObject`, but the functions are `ibase` and `ibase_mut` respectively. Perhaps we should rename those to `raw_interactive` and `raw_interactive_mut` to match...

Reply:
Good idea, will do :)